### PR TITLE
Fixed phpdoc annotations

### PR DESCRIPTION
**Pull request by chapay**

### CODE REVIEW

**Comment by Spomky** [comment]
+1

File: Event/OAuthEvent.php [hunk 1]
```diff
@@ -37,7 +37,9 @@ class OAuthEvent extends Event
     private $isAuthorizedClient;
 
     /**
-     * @param UserInterface $user
+     * @param UserInterface   $user
+     * @param ClientInterface $client
+     * @param Boolean         $isAuthorizedClient
```
Comment (Ener-Getick) [inline Event/OAuthEvent.php]: please use ``bool`` here instead.

Comment (chapay) [inline Event/OAuthEvent.php]: Done.
I just used the same CS which is already used in `OAuthEvent.php` and [ClientInterface.php](https://github.com/FriendsOfSymfony/FOSOAuthServerBundle/blob/master/Model/ClientInterface.php#L35) files.

Comment (Ener-Getick) [inline Event/OAuthEvent.php]: I see but I prefer following the symfony standards 😉

**Comment by Ener-Getick** [comment]
Thank you @chapay.

**Pull request merged**